### PR TITLE
Brady doc

### PR DESCRIPTION
**Pull request by engband**
Passed all unit tests.

edits to documentation and a few other files, fixing typos, updating old info, etc.

### CODE REVIEW

File: docs/user-guide/parameter-configuration.md [hunk 1]
```diff
@@ -57,39 +57,39 @@ rosservice call /param_write
...
 
-| Parameter | Description | Type | Fixed Wing Value
+| Parameter | Description | Type | Fixed-Wing Value
 |-----------|-------------|------|---------------|
 | MOTOR_PWM_UPDATE | Refresh rate of motor commands to motors and servos (Hz) - See motor documentation | int |  50 |
 | ARM_SPIN_MOTORS | Enforce MOTOR_IDLE_PWM | int |  false |
 | MOTOR_IDLE_THR | min throttle command sent to motors when armed (Set above 0.1 to spin when armed) | float |  0.1 |
 | ARM_CHANNEL | RC switch channel mapped to arming [0 indexed, -1 to disable] | int |  4 |
-| FIXED_WING | switches on passthrough commands for fixedwing operation | int |  true |
+| FIXED_WING | switches on passthrough commands for fixed-wing operation | int |  true |
```
Comment (superjax) [inline docs/user-guide/parameter-configuration.md]: The parameter file is auto-generated from `params.cpp`.  These changes will get overwritten if we don't make the changes there.

**Comment by dpkoch** [comment]
Note to self: Fix the missing "sil_diagram.svg" error (cherry-pick 3d2360c)

**Comment by dpkoch** [comment]
We should also add a not saying that we recommend F4 boards because F1 boards will not be supported going forward

File: docs/developer-guide/building-flashing.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ To build only the F4 firmware, use `make BOARD=REVO`. To build only the F1 firmw
 
 ## Flashing Newly-Built Firmware
 
-First, make sure you have configured your computer as described in the [Serial Port Configuration](/user-guide/flight-controller-setup/#serial-port-configuration) section of the user guide.
+First, make sure you have configured your computer as described in the [Serial Port Configuration](../user-guide/flight-controller-setup.md#serial-port-configuration) section of the user guide.
```
Comment (engband) [inline docs/developer-guide/building-flashing.md]: curious as to why this change was made, since the browser interprets it already as
http://127.0.0.1:8000/user-guide/flight-controller-setup/#serial-port-configuration
which is what it used to be. Is this a 'best practices', preference, or something else?
Similar changes were made in User-Guide: Getting Started. I don't mean to come across as rude, just want to learn.

Comment (dpkoch) [reply docs/developer-guide/building-flashing.md]: Yeah it's just a best practices thing; that's how the mkdocs documentation says to do internal links
